scripts/trigger_oozie_job.sh

```diff
@@ -35,11 +37,14 @@ echo "Successfully transfered ./configuration/${ENV}/job.properties to bi-${ENV}
 
 # Trigger the oozie job and get the job id, remove unused chars from the id and then poll it.
 # JOB_ID is something like 'job: 213871982-213123123-asdasd', we remove 'job: '
-ssh -tt bi-${ENV}-ci@${HOST} OOZIE_HOME=$OOZIE_HOME ENV=$ENV 'bash -s' << 'ENDSSH'
+ssh -tt bi-${ENV}-ci@${HOST} OOZIE_HOME=$OOZIE_HOME ENV=$ENV INDEX_NAME=$INDEX_NAME 'bash -s' << 'ENDSSH'
     TIMEOUT=1000
     INTERVAL=1
     OOZIE_ID_INDEX=5
 
+    export ENVIRONMENT=$ENV
+    export INDEX_NAME=$INDEX_NAME
```

Comment:
Why are these exported? It's not immediately obvious here

Reply:
I've added a comment explaining them, basically Oozie reads the `job.properties` file that is SCP'd over and it needs to be able to use the index name that is generated by a previous step in the `Jenkinsfile`. 

`ENV` is required by other shell scripts that make up part of the Oozie workflow, which may not be needed as the username (bi-dev/test/beta) indicates this. It is probably out of scope for this task to modify the actual Oozie workflow so I'll keep it in mind for when future changes are made to the workflow.

Reply:
Thanks for the comment, but I still don't understand whats happening. You are setting the environment variables for the current session, not the one which the oozie job runs in AFAIK.

Reply:
Ah, I see what you mean. The previous pipeline stores the `job.properties` file in Jenkins, so variables are replaced there before being SCP'd.
I have changed the solution to just `sed` the `INDEX_NAME` into the file.

Reply:
:100: this is a muuuchhh nicer